pkg/httpd/simpleserver.go

```diff
@@ -48,12 +49,26 @@ func (simpleServer *SimpleServer) Start() error {
 	mux.HandleFunc("/healthz", webhook.HealthCheckHandler)
 	mux.HandleFunc("/mutate", admissionHandler.HandleAdmission)
 
+	addr := ":9090"
```

Comment:
this should definitely be configurable. It should probably be in `SimpleServer` as `MetricsPort`.

We should also not start a separate server unless that port is specifically specified. i.e.

```
if MetricsPort != nill && MetricsPort != Port {
  go startMetricsServer
} else {
  mux.Handle("/metrics", promhttp.Handler())
}
```

You will also need to add it to the helm chart.

Reply:
updated